mpas_analysis/ocean/climatology_map_waves.py

```diff
@@ -0,0 +1,528 @@
+        fields = [field for field in fields if field['prefix'] in fieldList]
+
+        variableList = [field['mpas'] for field in fields] 
+
+        makeTables = config.getboolean(sectionName, 'makeTables')
+        
+
+        if makeTables:
+            #for regionGroup in regionGroups:
+            for season in seasons:
+                tableSubtask = WavesTableSubtask(
+                    parentTask=self,
+                    mpasClimatologyTask=mpasClimatologyTask,
+                    controlConfig=controlConfig,
+                    regionMasksTask=regionMasksTask,
+                    fields=fields,
+                    season=season) 
+                self.add_subtask(tableSubtask)
+
+
+        # the variableList variables  will be added to
+        # mpasClimatologyTask along with the seasons.
+        #remapClimatologySubtask = RemapMpasClimatologySubtask(
+        remapClimatologySubtask = RemapMpasWavesClimatology(
+            mpasClimatologyTask=mpasClimatologyTask,
+            parentTask=self,
+            climatologyName='wave',
+            variableList=variableList,
+            comparisonGridNames=comparisonGridNames,
+            seasons=seasons,
+            iselValues=None)
+
+
+        for field in fields:
+
+            fieldPrefix = field['prefix']
+            refFieldName = field['mpas']
+            upperFieldPrefix = fieldPrefix[0].upper() + fieldPrefix[1:]
+            configSectionName = '{}{}'.format(self.taskName, upperFieldPrefix)
+
+            if controlConfig is None:
+
+                galleryName = field['titleName']
+                refTitleLabel = None 
+                diffTitleLabel = None 
+
+            else:
+                controlRunName = controlConfig.get('runs', 'mainRunName')
+                refTitleLabel = f'{field["titleName"]} (Control: {controlRunName})'
+                diffTitleLabel = 'Main - Control'
+
+            outFileLabel = '{}Wave'.format(fieldPrefix)
+
+            for comparisonGridName in comparisonGridNames:
+                for season in seasons:
+
+                    subtaskName = 'plot{}_{}_{}'.format(upperFieldPrefix,
+                                                        season,
+                                                        comparisonGridName)
+
+                    # make a new subtask for this season and comparison grid
+                    subtask = PlotClimatologyMapSubtask(
+                        parentTask=self, 
+                        season=season,
+                        comparisonGridName=comparisonGridName,
+                        remapMpasClimatologySubtask=remapClimatologySubtask,
+                        controlConfig=controlConfig,
+                        subtaskName=subtaskName)
+
+
+                    subtask.set_plot_info(
+                        outFileLabel=outFileLabel,
+                        fieldNameInTitle=field['titleName'],
+                        mpasFieldName=field['mpas'],
```

Comment:
This would need to be fixed.  If we rename these variables as part of masking, we can use the prefix name.
```suggestion
                        mpasFieldName=field['prefix'],
```